tlatools/org.lamport.tlatools/test/tla2sany/semantic/TestLevelChecking.java

```diff
@@ -77,14 +88,79 @@ public String summarize(Errors log) {
      * this hypothesis is true. For further info, see this issue:
      * https://github.com/tlaplus/tlaplus/issues/1008
      */
-    new LevelCheckingTestCase("x == 5.foo", true),
+    new LevelCheckingTestCase("c.foo",           true),
```

Comment:
Why does the expression change? Additionally, there appears to be unnecessary indentation.